pkgs/tools/networking/airgeddon/default.nix

```diff
@@ -0,0 +1,126 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+  # Required
+, aircrack-ng
+, bash
+, gawk
+, iproute2
+, iw
+, pciutils
+, procps
+, tmux
+  # Optionals
+  # Missing in nixpkgs: beef, hostapd-wpe, asleap
+, bully
+, crunch
+, dhcp
+, dnsmasq
+, ettercap
+, hashcat
+, hcxdumptool
+, hcxtools
+, hostapd
+, lighttpd
+, mdk4
+, nftables
+, openssl
+, pixiewps
+, reaverwps-t6x # Could be the upstream version too
+, wireshark-cli
+  # Support groups
+, supportWpaWps ? true # Most common use-case
+, supportHashCracking ? false
+, supportEvilTwin ? false
+}:
+with lib;
```

Comment:
What is exactly the reason for this with lib?

Reply:
In general we prefer not to use top-level `lib` because this polutes the global namespace. Also `with` is kinda problematic for performance reasons.